src/main/java/seedu/address/logic/parser/AddCommandParser.java

```diff
@@ -54,32 +55,45 @@ public AddCommand parse(String args) throws ParseException {
             }
 
             Name name = ParserUtil.parseName(argMultimap.getValue(PREFIX_NAME).get());
+
             Phone phone = ParserUtil.parsePhone(argMultimap.getValue(PREFIX_PHONE).get());
+
             Gender gender = ParserUtil.parseGender(argMultimap.getValue(PREFIX_GENDER).get());
+
             Qualification qualification = ParserUtil.parseQualification(
                     argMultimap.getValue(PREFIX_QUALIFICATION).get());
+
             Remark remark;
+            // Handles remark if it exists
             if (!arePrefixesPresent(argMultimap, PREFIX_REMARK)) {
                 remark = new Remark();
             } else {
                 remark = ParserUtil.parseRemark(argMultimap.getValue(PREFIX_REMARK).get());
             }
+
             Set<Tag> tagList = ParserUtil.parseTags(argMultimap.getAllValues(PREFIX_TAG));
 
             Tutor tutor = new Tutor(name, phone, gender, qualification, remark, tagList);
             return new AddCommand(tutor, PersonType.TUTOR);
             // No break necessary due to return statement
         case STUDENT:
```

Comment:
The parse method is over 30 lines long, you might want to abstract out some parts so you can have better SLAP?

Reply:
Actually I think the 30 lines long thing has nothing to do with SLAP..? 😂 SLAP is about the same level of abstraction within a code fragment, but shortening the method length is good for readability~